guides/advanced_guides/faceted_search.md

```diff
@@ -48,27 +50,42 @@ $ curl \
 
 When performing a search, you can specify:
 
-1. The facets to filter on with the facet syntax (explained below), thanks to the optional parameter `facetFilters`.
-2. the facets for which to retrieve the matching count for, with the `facets` optional parameter.
+### The facets for which to retrieve the matching count
 
-## Additional fields
+`facets=[<facetName>, <facetName>, ...]`
 
-The result of the search query contains two additional fields if the `facets` parameter has been set:
+This attribute can take two values:
 
-  1. `Facets`: returning a count of candidates for each requested facets in the facets parameter, if set to `[*]` a count for all facets is returned.
-  2. `ExhaustiveFacetsCount` telling whether the above count is exhaustive or approximative.
+- `[<facetName>, <facetName>, ...]` (Optional, array of strings, defaults to `null`)
 
-The syntax for passing arguements through facetFilters is `["facetName:facetValue"] or [["FacetName:FacetValue"]]`
+  Comma-separated list of facets for which to retrieve the matching count. The number of remaining candidates for each specified facet is returned. If a facet name doesn't exist, it will be ignored.
 
-### Example
+- `["*"]`
+
+  The `*` character can also be used. In that case, a count for all facets is returned.
+
+If the `facets` parameter has been set, the returned results will contain two additional fields:
+
+1. `facets`: The number of remaining candidates for each specified facet.
+
+2. `exhaustiveFacetsCount`: Whether the above count is **exhaustive** or **approximative**.
```

Comment:
We should enumerate the possible values it can take (i.e. `true` or `false`) :)

Reply:
Thanks!